libsplinter/src/rest_api/mod.rs

```diff
@@ -859,22 +859,28 @@ impl RestApiBuilder {
                             ));
                         }
 
+                        #[cfg(feature = "biome-oauth")]
```

Comment:
This change seems like it should be in its own commit.

Reply:
No, this change was necessary because the ownership changed (see the above `into_iter` change).  The feature guard is due to the fact that it was previously only used inside a later block that was guarded by that feature.